app/templates/document_download_template.html

```diff
@@ -26,8 +26,12 @@
   {% endblock %}
 {% endblock %}
 
-{% block pageTitle %}
-   GOV.UK - Document download
+{% set _per_page_title %}
+  {% block per_page_title %}Document Download{% endblock %}
+{% endset %}
+
+{% block page_title %}
```

Comment:
This needs to be camelCased to match the blocks set by GOVUK Frontend.
```suggestion
{% block pageTitle %}
```

Reply:
Fixed and squashed into 5ee128e